projects/angular/src/data/datagrid/datagrid-filter.ts

```diff
@@ -145,8 +147,29 @@ export class ClrDatagridFilter<T = any>
     return !!this.filter && this.filter.isActive();
   }
 
+  toggleButtonAriaLabel: string;
+
+  ngOnChanges() {
+    this.setToggleButtonAriaLabel();
+  }
+
   override ngOnDestroy(): void {
     super.ngOnDestroy();
     this.subs.forEach(sub => sub.unsubscribe());
   }
+
+  /**
+   * This is not in a getter to prevent "expression has changed after it was checked" errors.
+   * And it's more performant this way since it only runs on change.
+   */
+  private setToggleButtonAriaLabel() {
```

Comment:
Some time ago we were trying to keep Clarity working in SSR scenario. I think we already diverged from this goal. But it may be worth talking about it and evaluating if we want to support it and what the cost will be.

Reply:
I'll look into that.